.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+<!-- Enter a brief description/summary of your PR here. What does it fix/what does it change/how was it tested (even manually, if necessary)? -->
+## Summary of the Pull Request
+
+<!-- Other than the issue solved, is this relevant to any other issues/existing PRs? --> 
+## References
+
+<!-- Please review the items on the PR checklist before submitting-->
+## PR Checklist
+[ ] Closes #xxx
```

Comment:
I'm pretty sure if you prepend these lines with a `* `, then they'll show up as checkboxes in the rendered markdown.

* [ ] Like this

```markdown
* [ ] Like this
```